main.tf

```diff
@@ -3,29 +3,21 @@ This file is used to implement the ROOT module.
 *********************************************************************/
 
 resource "ibm_db2" "db2" {
-  name              = var.db2_instance_name
-  resource_group_id = var.resource_group_id
-  location          = var.region
-  plan              = var.subscription_id == null ? "performance" : "PerformanceSubscription"
-  service           = "dashdb-for-transactions"
-  high_availability = var.enable_high_availability ? "yes" : "no"
-  service_endpoints = var.service_endpoints
-
-  parameters = {
-    "oracle_compatibility" : var.enable_oracle_compatibility ? "yes" : "no",
-    "version" : "12",
```

Comment:
what happened to version? No longer configurable by user?

Reply:
Yes, from https://registry.terraform.io/providers/IBM-Cloud/ibm/1.73.0/docs/resources/db2_instance on, version is not one of the variables that can be passed